addons/mrp/models/stock_warehouse.py

```diff
@@ -101,6 +103,8 @@ def _get_route_name(self, route_type):
 
     def _get_global_route_rules_values(self):
         rules = super(StockWarehouse, self)._get_global_route_rules_values()
```

Comment:
Please as you know `s / super(StockWarehouse, self). / super().`

and in all code below. Thanks